crates/blockchain/mempool.rs

```diff
@@ -79,6 +79,14 @@ pub fn filter_transactions(
         if filter.only_plain_txs && is_blob_tx || filter.only_blob_txs && !is_blob_tx {
             return false;
         }
+
+        // This is a temporary fix to avoid invalid transactions to be included.
+        // This should be removed once https://github.com/lambdaclass/ethereum_rust/issues/680
+        // is addressed.
+        if tx.effective_gas_tip(filter.base_fee).is_none() {
+            return false;
+        }
```

Comment:
We could add this check on the `else` block of the min_tip check, that way we can leave the check even when the issue is fixed

Reply:
But, this should be an `else if` right? We still need to know if `effective_gas_tip` returned `None`, or am I missing something?

Reply:
As I understood it, yes, we still need to check for the `None` of `effective_gas_tip` even in the else of `let Some(min_tip) = filter.min_tip`

Reply:
Gotcha

Reply:
Done [62e4d9c](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1018/commits/62e4d9c11fe53c6cb79bc58192b818c304cd6f01)